featurebyte/models/feature_list_store_info.py

```diff
@@ -242,13 +247,17 @@ def get_feature_specs_definition(
             base_dataframe_schema,
             pyspark_import_statement,
         ) = self._get_base_dataframe_schema_and_import_statement(target_spec=target_spec)
+
+        exclude_columns = self.exclude_columns
+        if skip_exclude_columns:
```

Comment:
Is it possible to also add any columns in `skip_exclude_columns` into the `schema` if it's not a feature?

Reply:
This parameter is only used to remove the columns from the original exclude list in the template. If we want to add other custom columns to schema, we need to have at least 2 information (column name & column type). That means we likely need another parameter to do this.

Since the most complicated part of the script is the feature specs. Would prefer user to do something like this:
```
feature_specs = deployment.databricks.get_feature_specs()

exclude_columns = [
    "POINT_IN_TIME",
    "__feature_V240103__part0",
    "__feature_V240103__part1",
    "__featurebyte_dummy_entity",
    "__relative_frequency_V240103__part0",
    "__relative_frequency_V240103__part1",
    "__req_col_feature_V240103__part0",
    "transaction_id",
]
# Prepare the dataset for log model
# 'features' is a list of feature lookups to be included in the training set
# 'exclude_columns' is a list of columns to be excluded from the training set
target_column = "float_target"
schema = StructType(
    [
        StructField(target_column, DoubleType()),
        StructField("__featurebyte_dummy_entity", StringType()),
        StructField("transaction_id", LongType()),
        StructField("POINT_IN_TIME", TimestampType()),
    ]
)
spark = SparkSession.builder.getOrCreate()
log_model_dataset = fe.create_training_set(
    df=spark.createDataFrame([], schema),
    feature_lookups=features,
    label=target_column,
    exclude_columns=exclude_columns,
)
```
Though user needs to write more codes, but it is much clearer to the user.

Reply:
indeed. I forgot we need the dtype as well.